src/pages/home/report/ReportActionCompose.js

```diff
@@ -929,7 +929,7 @@ class ReportActionCompose extends React.Component {
                     style={this.props.isComposerFullSize ? styles.chatItemFullComposeRow : {}}
                     contentContainerStyle={this.props.isComposerFullSize ? styles.flex1 : {}}
                 >
-                    {shouldShowReportRecipientLocalTime && <ParticipantLocalTime participant={reportRecipient} />}
+                    {shouldShowReportRecipientLocalTime && !!reportRecipient && <ParticipantLocalTime participant={reportRecipient} />}
```

Comment:
Sure @rushatgabhane , the other way is to create a function to check if `reportRecipient` exists, since I can't put it inline without the `!!`. I saw the `!!` used in another place too in the code, that's why I used it here.

Reply:
Can you just use `Boolean(reportRecipient)`?

Reply:
Hey @jasperhuangg , the reason I don't use `Boolean(reportRecipient)` is because an empty Object will come up as `true` in Boolean. I believe in the case of an empty object we still don't want to show it, correct me if I am wrong though.